app/src/main/java/org/fossasia/openevent/general/auth/ProfileFragment.kt

```diff
@@ -42,13 +42,22 @@ class ProfileFragment : Fragment() {
     private val EMAIL: String = "EMAIL"
 
     private fun redirectToLogin() {
-        findNavController(rootView).navigate(R.id.loginFragment, null, getAnimSlide())
+        val args = getString(R.string.log_in_first)
+        val bundle = bundleOf("snackbar_message" to args)
```

Comment:
No use of the constant unless it is used everywhere

Reply:
Try to understand the reason why the changes are requested rather than blindly making them